src/effects/nyquist/Nyquist.h

```diff
@@ -240,7 +240,7 @@ class AUDACITY_DLL_API NyquistEffect final : public Effect
    TranslatableString mCopyright;
    wxString          mManPage;   // ONLY use if a help page exists in the manual.
    wxString          mHelpFile;
-   bool              mHelpFileExists;
+   mutable bool      mHelpFileExists;
```

Comment:
This is the one place where you added mutable.

Reply:
It is mutable because it is reassigned in `NyquistEffect::HelpPage`.  But it would be better to find a fix that does not rely on mutable.  Instead, mHelpFileExists should be assigned at some earlier place inside a non-const function that initializes the NyquistEffect.

What other things does the correct value of mHelpFileExists depend on?  When are those things assigned and no longer changed, so that you could move the computation of mHelpFileExists correctly to an earlier place that is not too early?

Reply:
I moved the intialization of mHelpFileExists at the end of the constructor, please see commit 3aa08b2